configure-alertmanager-operator/configure-alertmanager-operator.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: configure-alertmanager-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```

Comment:
We would want to change the name of this to sss-artifacts-template.yaml ?

Reply:
Maybe? I thought about it, but wanted to keep it consistent.

Reply:
fwiw we don't have a preference from the app-sre side of things